tests/unit/commands/delete/test_delete_context.py

```diff
@@ -149,14 +121,12 @@ def test_delete_no_user_input(
     def test_delete_context_valid_execute_run(self, get_boto_client_provider_mock, patched_click_get_current_context):
         patched_click_get_current_context = MagicMock()
         with DeleteContext(
-            stack_name=None,
+            stack_name="test",
```

Comment:
Did something else get read in for these values to `test`?

Reply:
Yea it was reading in some values from a config file somewhere, but since all that happens outside of the context now i had to set the values here